### PR TITLE
Requests functions

### PR DESCRIPTION
**Pull request by MCJOHN974**
@yegor256 here I add ```functions.c``` with counting of requests to carts. Output is similar in ```objects``` and ```functions``` and, i think, i remove procedural style.

### CODE REVIEW

File: objects.cpp [hunk 1]
```diff
@@ -55,14 +55,23 @@ class Tangible : public Stocked {
 };
 
 class Cart {
+    static int requests;
```
Comment (yegor256) [inline objects.cpp]: @MCJOHN974 why making it static instead of a global variable, which we already have?

**Comment by yegor256** [comment]
@MCJOHN974 static attributes are a very bad idea in OOP. Let's do global vars instead.

**Comment by MCJOHN974** [comment]
@yegor256 why static is bad idea, can you explain, please

**Comment by yegor256** [comment]
@MCJOHN974 sure, read this: https://www.yegor256.com/2014/05/05/oop-alternative-to-utility-classes.html Static methods/variables are in general a bad idea in OOP. Global ones too, but we already have them: https://www.yegor256.com/2018/07/03/global-variables.html

**Comment by MCJOHN974** [comment]
Thanks

**Comment by MCJOHN974** [comment]
@yegor256 here i remove static from everywere. Now ```functions.a``` and ```objects.a``` have similar output and functions works much faster

**Comment by yegor256** [comment]
@MCJOHN974 looks good now, but I can't merge this branch to due merge conflicts. Please, resync your branch with `master.

**Comment by MCJOHN974** [comment]
@yegor256 in last commit i pull yours ```master``` and merge it into this branch

**Comment by MCJOHN974** [comment]
@yegor256 so, i think now everything is ready for merge

**Comment by yegor256** [comment]
@MCJOHN974 thanks!

**Pull request merged**